lib/plugins/wallet/cardano/cardano.js

```diff
@@ -0,0 +1,85 @@
+const { walletId } = require('./mnemonic')
+const cardanoApi = require('./client')
+
+// TODO: work with BN here instead of api.js?
+
+async function checkCryptoCode(cryptoCode) {
+  if (cryptoCode !== 'ADA') {
+    return Promise.reject(new Error(`Unsupported crypto: ${cryptoCode}`))
+  }
+
+  return Promise.resolve()
+}
+
+/**
+ * TODO: instead of using walletId function we can
+ *       inject walletId/index into account object
+ *       in wallet.js - it's better?
+ */
+async function balance(account, cryptoCode) {
+  return checkCryptoCode(cryptoCode)
+    .then(() => walletId())
+    .then(({ walletId }) => cardanoApi.walletBalance(walletId, 1))
+}
+
+async function sendCoins(account, address, cryptoAtoms, cryptoCode) {
```

Comment:
`account` is unused. Is that intended? Same above ...

Reply:
yes, it's needed header for this "common" functions  (same for BTC, etc)

Reply:
So please just add some comment, explaining why it is not used, so in the future, there is no need to think about it.